packages/scripts/scripts/format-js.js

```diff
@@ -104,7 +104,7 @@ if ( fileArgs.length === 0 ) {
 
 // Converts `foo/bar` directory to `foo/bar/**/*.js`
 const globArgs = dirGlob( fileArgs, {
-	extensions: [ 'js', 'jsx', 'ts', 'tsx' ],
+	extensions: [ 'js', 'jsx', 'ts', 'tsx', 'yml', 'yaml' ],
```

Comment:
It's `format-js` so I don't think it should be concerned about YAML. Can we create another script that would cover config formats like YAML or JSON?

Reply:
Yeah, I was worried that that could be of concern 😅 

> Can we create another script that would cover config formats like YAML or JSON?

IDK, where would we draw the line? We're already using the same script for `.ts(x)`; I think at least JSON would also be "close enough" to JS to be included.

I get that YAML is quite different, but the drawback is that we'd end up with two separate scripts that do pretty much the exact same thing -- running `prettier` on a bunch of files. Which can be done -- potentially by extracting some logic into a shared module -- but it's going to be more than a one-line change as here 😬 

Would we consider renaming `format-js` instead? 🤔

Reply:
I don't think we should consider the number of lines to update as a deciding factor. We need to keep in mind backward compatibility and flexibility of usage. At the moment, it'd be a breaking change for those who use `format-js` because it suddenly would format more files than before. What if someone wants to opt-out of it?

Reply:
what if we rename the command just `format` ?

Reply:
FYI, in Calypso we moved to `eslint` as the "format engine", used to format all types of files: `.js`, `.ts`, `.md`... It uses `prettier` as a plugin (so you get the same formatting you'd get with just `prettier`), plus it allows us to use all `eslint` goodies like custom rules, disable rules per line/file, custom reporters...

Reply:
We can keep an alias for backward compatibility and rename, but still, the question remains, what if someone wants to format only JS or only YAML? If we sort it out, then everything is fine.

Reply:
>  what if someone wants to format only JS or only YAML? If we sort it out, then everything is fine.

Why would someone want that? I mean it's definitely possible but 95% of the cases, people just want to have guidelines for all their code base and `wp-scripts` is about a good opinionated setup so I'm personally in favor of removing all configs and possibilities from folks and just simplifying everything. `format` command seems to go in that direction for me.

Reply:
An alias is a good idea to avoid distruption but we can also make a major version change. If I had to choose I'd do both  and deprecate the old.

Reply:
Agree with Riad. IMO, this approach also better reflects the idea of `prettier`: It takes care of formatting for you so you don't have to worry about _how_ exactly you want to fine-tune whitespace etc -- going so far as to even include new file formats, once it supports them. (Meaning that if you would run `prettier` directly, I think it'd also just start formatting those new files unless you explicitly excluded them.)

We could make it a breaking change: Bump `wp/scripts` to a new major version, and rename `format-js` to `format`.

Reply:
You can give it a try to add `format` that covers all files that Prettier supports. I hope it is compatible with `stylelint` for SCSS files.

For `format-js`, it would be mostly about the convenience of developers who already have it integrated into the project. `npx wp-create-block` uses `wp-scripts format-js` so it might impact a larger group. We can remove it from the documentation and replace it with `format`. It would be simply nice to keep an alias with a deprecation warning.

Reply:
> It would be simply nice to keep an alias with a deprecation warning.

Sounds good to me!

Reply:
I've changed the script name and added a deprecation warning per the latest commits.